app/models/folio/pickup_destination.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Folio
+  # Encapsulates pickup destination logic for FOLIO service points
+  class PickupDestination
+    # Get the default destination
+    def self.default_destination
+      Settings.folio.default_service_point
+    end
+
+    # @param code: service point or library code
+    def initialize(destination_code)
+      @code = destination_code
+      @type = 'ServicePoint'
```

Comment:
I don't see `@type` used. Do we need it?

Reply:
I removed it